fuzz/src/router.rs

```diff
@@ -32,6 +32,7 @@ use utils::test_logger;
 use std::collections::HashSet;
 use std::sync::Arc;
 use std::sync::atomic::{AtomicUsize, Ordering};
+use std::convert::TryInto;
```

Comment:
Not used.

Reply:
Removed in https://github.com/lightningdevkit/rust-lightning/pull/1286/commits/f5b1b46c43d740cf531f9f4ae669303043680cf0.